File: Sem- 1/C/While Loop/factors.c
#include<stdio.h>
void main(){
	printf("Factors of a Number");
	int n,i=1;
	printf("\nEnter any number: ");
	scanf("%d",&n);
	printf("Factors of %d: ",n);
	while(i<=n){
		if(n%i==0){
			printf("\n\t%d",i);
			i++;
		}
		else{
			i++;
		}
	}
}

File: Sem- 3/Data Structure/Array/Factors.java
import java.util.Scanner;
public class Factors{
	public static void main(String[] args){
		Scanner sc=new Scanner(System.in);
		System.out.println("Factors of a Number");
		System.out.print("Enter any number: ");
		int n=sc.nextInt();
		System.out.print("Factors of "+n+": ");
		for(int i=1;i<=n;i++){
			if(n%i==0){
				System.out.print(i+" ");
			}
		}
	}
}